packages/cozy-doctypes/src/File.spec.js

```diff
@@ -233,6 +237,27 @@ describe('File model', () => {
     })
   })
 
+  describe('generateFileNameForRevision', () => {
+    beforeEach(() => {
+      MockDate.set(MOCKED_DATE)
+    })
+
+    afterEach(() => {
+      jest.restoreAllMocks()
+      MockDate.reset()
+    })
+    it('should generate the right file name for a revision', () => {
+      const expectedFilename = 'test_01 January - 12h00.pdf'
+      const date = new Date(MOCKED_DATE)
```

Comment:
Nit: I am not sure why you MockDate and then reset it, since you pass directly the date to `new Date`, it does not seem necessary.